### PR TITLE
docs: document new Flagd CRD

### PR DESCRIPTION
**Pull request by bacherfl**
This PR documents the recently introduced `Flagd` CRD

### CODE REVIEW

File: docs/flagd.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# Flagd
+
+The `Flagd` version `v1beta1` CRD is used to create a standalone flagd deployment,
```
Comment (thisthat) [inline docs/flagd.md:3]:
```suggestion
The CRD `Flagd` at version `v1beta1` is used to create a standalone flagd deployment,
```

File: docs/flagd.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+accompanied by a `Service` and an optional `Ingress` to expose its API
+endpoint to clients outside the cluster.
+
+Handling this resource can be enabled/disabled by setting `managerConfig.flagdResourceEnabled` helm value of the operator
```
Comment (thisthat) [inline docs/flagd.md:7]:
```suggestion
The handling this resource can be enabled/disabled by setting `managerConfig.flagdResourceEnabled` Helm value of the operator
```

**Review by thisthat — APPROVED** [review]

**Review by toddbaert — COMMENTED** [review]
@bacherfl this makes sense to me. I ran everything locally though and I was a bit suprised.... as indicated by the tests, the version of flagd used is `v0.9.0` (the [SidecarTag](https://github.com/open-feature/open-feature-operator/blob/d54914460b9f01e10bdc958a46ff210fd0f4c374/common/types/envconfig.go#L27)) not the [FlagdTag](https://github.com/open-feature/open-feature-operator/blob/d54914460b9f01e10bdc958a46ff210fd0f4c374/common/types/envconfig.go#L15).

**Comment by bacherfl** [comment]
> @bacherfl this makes sense to me. I ran everything locally though and I was a bit suprised.... the version of flagd used by default is `v0.9.0` (the [SidecarTag](https://github.com/open-feature/open-feature-operator/blob/d54914460b9f01e10bdc958a46ff210fd0f4c374/common/types/envconfig.go#L27)) not the [FlagdTag](https://github.com/open-feature/open-feature-operator/blob/d54914460b9f01e10bdc958a46ff210fd0f4c374/common/types/envconfig.go#L15).
> 
> This causes some practical problems since 0.9.0 didn't support the `sync` or `ofrep` protocols. I can't immediately see why this seems to be happening.
> 
> version [ version ](https://private-user-images.githubusercontent.com/25272906/332963856-c23e2ee1-8505-41f9-9754-f4d33f2cd09a.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0Mzk5NjAsIm5iZiI6MTcxNjQzOTY2MCwicGF0aCI6Ii8yNTI3MjkwNi8zMzI5NjM4NTYtYzIzZTJlZTEtODUwNS00MWY5LTk3NTQtZjRkMzNmMmNkMDlhLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTIzVDA0NDc0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4YjRmNzI2MDJjMDFmN2M5MjM0ZjhhYWNlNTkxNjRiYWM2MWQ1MjY3MDY1OGQ1OTVjNGJmMzBiMWMwNzY3NzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gJ-c922I6GFGwKILOJQ-UQj9aqJ6RSWYZRzHTDu4EOk) [ ](https://private-user-images.githubusercontent.com/25272906/332963856-c23e2ee1-8505-41f9-9754-f4d33f2cd09a.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0Mzk5NjAsIm5iZiI6MTcxNjQzOTY2MCwicGF0aCI6Ii8yNTI3MjkwNi8zMzI5NjM4NTYtYzIzZTJlZTEtODUwNS00MWY5LTk3NTQtZjRkMzNmMmNkMDlhLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTIzVDA0NDc0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4YjRmNzI2MDJjMDFmN2M5MjM0ZjhhYWNlNTkxNjRiYWM2MWQ1MjY3MDY1OGQ1OTVjNGJmMzBiMWMwNzY3NzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gJ-c922I6GFGwKILOJQ-UQj9aqJ6RSWYZRzHTDu4EOk)

thanks, I will look into this today

**Comment by thisthat** [comment]
This has been address with https://github.com/open-feature/open-feature-operator/pull/643 :)

**Review by toddbaert — APPROVED** [review]

**Pull request merged**